### PR TITLE
Add SCALE function block for signal processing.

### PR DESCRIPTION
**Pull request by franz-hoepfinger-4diac**
- Added a new SCALE function block for scaling signals in the SIGNALPROCESSING module.
- Included necessary header and source files for the SCALE function block implementation.

### CODE REVIEW

File: src/modules/SIGNALPROCESSING/SCALE_fct.cpp [hunk 1]
```diff
@@ -0,0 +1,155 @@
+/*************************************************************************  
+ *** Copyright (c) 2023 HR Agrartechnik GmbH
```
Comment (azoitl) [inline src/modules/SIGNALPROCESSING/SCALE_fct.cpp]: shouldn't this be 2024?

File: src/modules/SIGNALPROCESSING/SCALE_fct.h [hunk 2]
```diff
@@ -0,0 +1,97 @@
+/*************************************************************************  
+ *** Copyright (c) 2023 HR Agrartechnik GmbH
```
Comment (azoitl) [inline src/modules/SIGNALPROCESSING/SCALE_fct.h]: shouldn't this be 2024?

**Review by azoitl — APPROVED** [review]

**Comment by azoitl** [comment]
@franz-hoepfinger-4diac please resolve the conflict.

**Comment by franz-hoepfinger-4diac** [comment]
> @franz-hoepfinger-4diac please resolve the conflict.

done

**Pull request merged**